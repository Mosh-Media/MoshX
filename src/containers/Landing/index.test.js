import React from "react";
import LandingPage from "./index.js";
import renderer from "react-test-renderer";

it("Landing Page", () => {
  const tree = renderer.create(<LandingPage />).toJSON();
  expect(tree).toMatchSnapshot();
});
