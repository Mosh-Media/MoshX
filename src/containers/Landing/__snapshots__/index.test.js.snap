// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LandingPage Component should match the snapshot 1`] = `
<LandingPage
  classes={
    Object {
      "checkMark": "LandingPage-checkMark-4",
      "container": "LandingPage-container-1",
      "img": "LandingPage-img-3",
      "responsiveGrid": "LandingPage-responsiveGrid-2",
    }
  }
/>
`;
